web/sidebar_spec_supabase.js

```diff
@@ -9,13 +9,13 @@ module.exports = {
     {
       type: 'category',
       label: 'Auth',
-      items: ['reference/javascript/auth-signup', 'reference/javascript/auth-signin', 'reference/javascript/auth-signout', 'reference/javascript/auth-session', 'reference/javascript/auth-user', 'reference/javascript/auth-update', 'reference/javascript/auth-setauth', 'reference/javascript/auth-onauthstatechange', 'reference/javascript/auth-api-getuser', 'reference/javascript/reset-password-email'],
+      items: ['reference/javascript/auth-signup', 'reference/javascript/auth-signin', 'reference/javascript/auth-signout', 'reference/javascript/auth-session', 'reference/javascript/auth-user', 'reference/javascript/auth-update', 'reference/javascript/auth-setauth', 'reference/javascript/auth-onauthstatechange', 'reference/javascript/auth-api-getuser'],
```

Comment:
@dthyresson - any reason why this one was removed? `reference/javascript/reset-password-email'],`

I believe this is required for "Reset password" functionality?

Reply:
That was an accident as I thought it's details were duplicated in the yaml (there were multiple heading info sections for that method, I think). Thanks for catching. I'll restore it.

Reply:
Oh I see my error - I mistook it for a duplicate of the Auth Server only entry.

Reply:
@dthyresson does this one still need resolving?